econml/utilities.py

```diff
@@ -251,9 +268,12 @@ def cross_product(*XS):
     for X in XS:
         assert n == shape(X)[0]
 
+    # TODO: wouldn't making X1 vary more slowly than X2 be more intuitive?
```

Comment:
If we're never going to change it, maybe remove the TODO.

Reply:
done